File: CTDK_Python/144/_144.py
n = int(input("Nhap n: "))

dem = 0
i = 1
while i <= n:
    if n % i == 0:
        dem = dem + 1
    i = i + 1

if dem == 2:
    print("So nguyen to")
else:
    print("Khong la so nguyen to")

File: LTHAM/144/144.cpp
#include <iostream>
using namespace std;

int ktNguyenTo(int);

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	if (ktNguyenTo(n) == 1)
		cout << "So nguyen to" << endl;
	else
		cout << "Khong phai so nguyen to" << endl;
	return 0;
}

int ktNguyenTo(int n)
{
	int dem = 0;
	int i = 1;
	while (i <= n)
	{
		if (n % i == 0)
			dem++;
		i++;
	}
	if (dem == 2)
		return 1;
	return 0;
}
